docs/node.md

```diff
@@ -19,5 +19,22 @@ yarn add katex
 # or globally
 yarn global add katex
 ```
+
+## Building from Source
+
+To build you will need Git, Node.js 6.9 or later, and npm or Yarn.
+
+Clone a copy of the GitHub source repository:
+```bash
+git clone https://github.com/Khan/KaTeX.git
+cd KaTeX
+```
+
+Then install dependencies and run `build` script:
+```bash
+npm install # or yarn install
+npm run build # or yarn build
+```
 <br>
-> You can manually download the package from [here](https://github.com/Khan/KaTeX/releases)
+> You can manually download the package and source codes from
+[here](https://github.com/Khan/KaTeX/releases)
```

Comment:
"source codes" should be "source code".  Also it'd be nice to avoid a "here" link.